src/shared/components/errorContainer/ErrorContainer.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
```

Comment:
this will then be unnecessary

Reply:
Yes, I will use ErrorScreen rather than this.